rpc/client/localclient.go

```diff
@@ -41,13 +41,12 @@ func NewLocal(node *nm.Node) Local {
 	}
 }
 
-func (c Local) _assertIsClient() Client {
-	return c
-}
+var (
+	aLocal = Local{}
```

Comment:
why this?

Reply:
Because below, we'd end up creating more than one `Local{}`

Reply:
However, I'll remove it for clarity and instead use `Local{}` all the way.